tests/e2e/containers/containers.go

```diff
@@ -46,6 +46,15 @@ func NewManager(isUpgrade bool, isFork bool) (docker *Manager, err error) {
 	return docker, nil
 }
 
+// ExecTxCmd Runs ExecCmd, with flags for txs added.
+// namely `--chain-id={chain-id} -b=block --yes --keyring-backend=test`, and searching for `code: 0`
```

Comment:
Can we add --log-format=json to this? Its not added on all the CLI commands, but I assume this is not intentional